plugins/common/functions

```diff
@@ -316,12 +316,23 @@ release_and_deploy() {
       IMAGE_SOURCE_TYPE="dockerfile"
     fi
 
+    DOKKU_APP_SKIP_DEPLOY="$(dokku config:get $APP DOKKU_SKIP_DEPLOY || true)"
+    DOKKU_GLOBAL_SKIP_DEPLOY="$(dokku config:get --global DOKKU_SKIP_DEPLOY || true)"
+
+    DOKKU_SKIP_DEPLOY=${DOKKU_APP_SKIP_DEPLOY:="$DOKKU_GLOBAL_SKIP_DEPLOY"}
+
     dokku_log_info1 "Releasing $APP ($IMAGE)..."
     dokku release "$APP" "$IMAGE_SOURCE_TYPE" "$IMAGE_TAG"
-    dokku_log_info1 "Deploying $APP ($IMAGE)..."
-    dokku deploy "$APP" "$IMAGE_TAG"
-    dokku_log_info2 "Application deployed:"
-    dokku urls $APP | sed "s/^/       /"
+
+    if [ "$DOKKU_SKIP_DEPLOY" != "true" ]; then
```

Comment:
Sorry I missed this in the first pass, but following our [coding standards](https://github.com/progrium/bashstyle), can you swap out the deprecated `[` for the preferred `[[`? Thanks!